pkg/kapp/app/app_filter.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package app
+
+import "time"
+
+type AppFilter struct {
+	CreatedAtBeforeTime *time.Time
+	CreatedAtAfterTime  *time.Time
+
+	Labels []string
+}
+
+func (f AppFilter) Apply(apps []App) ([]App, error) {
+	var result []App
+
```

Comment:
Do we wanna add a check where we verify whether or not After time is less than Before time if both are specified or just show the user an empty table?

Reply:
Here we don't have something like `--before` and `--after` flags we just take `--filter-age` flag from user and this both `CreatedAtBeforeTime`  `CreatedAtAfterTime` getting used internally for filtering by age.

Reply:
Oh I see, makes sense 🚀